css/style.css

```diff
@@ -140,8 +153,7 @@ body {
   height: 68px;
   left: 375px;
   top: 495px;
-  background: rgb(237, 236, 255);
-  filter: blur(50px);
+  filter: blur(150px);
 }
 
 .social-icon {
```

Comment:
- [ ] [REQUIRED] Please note that the padding in-between the social icons is not as much as you have applied in your design. Kindly reduce the passing so that it can look like the one provided in the figma template.

Take a look at the font used for the `<h2>Multi-Post Stories</h2>`. Your font seems different, kindly stick to the design provided as much as possible(color, font, font-size etc).

Also, note that just after these social icons there is a little margin-bottom before the `<h2>My Recent Work</h2>`. You actually gave it some margin but it seems not to be enough, kindly check the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) to get the exact margin. Please click each image to have a better look between your implementation and the figma design.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![2](https://user-images.githubusercontent.com/58692590/150562157-af623fc4-8922-40ac-af50-cb1a69b0e03d.png) | ![22](https://user-images.githubusercontent.com/58692590/150562180-2362b0aa-dec5-4018-9aa8-6eecdd932b60.png).